onnxruntime/test/shared_lib/test_run_options.cc

```diff
@@ -12,7 +12,7 @@ TEST_F(CApiTest, run_options) {
   ASSERT_EQ(OrtRunOptionsSetRunTag(options.get(), "abc"), nullptr);
   ASSERT_STREQ(OrtRunOptionsGetRunTag(options.get()), "abc");
   ASSERT_EQ(OrtRunOptionsGetRunLogVerbosityLevel(options.get()), (unsigned)1);
-  ASSERT_EQ(OrtRunOptionsGetCacheFeedsFetchesInfoEnabled(options.get()), false);
-  OrtRunOptionsSetCacheFeedsFetchesInfoEnabled(options.get(), true);
-  ASSERT_EQ(OrtRunOptionsGetCacheFeedsFetchesInfoEnabled(options.get()), true);
+  ASSERT_EQ(OrtRunOptionsGetCacheFeedsFetchesInfoEnabled(options.get()), int(0));
```

Comment:
Just a coding style suggestion: you can either write static_cast<int>(0), or int(0), or (int)0.  But if we mix them in one file, even put them in adjacent lines, it's not beautiful.